fvwm/fvwm3.desktop.in

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=FVWM3
+Comment=The fvwm3 window manager
+Type=XSession
```

Comment:
This is incorrect.  If you run the `desktop-file-validate` program on the generated file, you'll see:

```
% desktop-file-validate ./fvwm/fvwm3.desktop

./fvwm/fvwm3.desktop.in: error: value "XSession" for key "Type" in group "Desktop Entry" is not a registered type value ("Application", "Link" and "Directory")
```

Therefore, the `Type` needs to be `Application` -- the installation path to `$(datarootdir)/xsessions` is still fien though.

Reply:
Interesting! That's what you get for just accepting some patch when "it works" (and, KDE's plasma workspace installs some `.desktop` file with the same invalid `Type` ...). But will be a very quick fix of course ;)